### PR TITLE
Added a new reports page for the 2019  And HTS Register 2019 WIP

### PR DESCRIPTION
**Pull request by dbaluku**
https://app.asana.com/0/1116981488119523/1129698612770038/f

### CODE REVIEW

File: omod/src/main/resources/apps/ugandaemr_reports_app.json [hunk 1]
```diff
@@ -39,6 +59,14 @@
         "url": "ugandaemrreports/reports.page",
         "icon": "icon-book",
         "order": 110
+      },{
+        "id": "ugandaemrreports2019.homepageLink",
+        "extensionPointId": "org.openmrs.referenceapplication.homepageLink",
+        "type": "link",
+        "label": "UgandaEMR Reports 2019",
```
Comment (ssmusoke) [inline omod/src/main/resources/apps/ugandaemr_reports_app.json]: @dbaluku 

1. Rename to UgandaEMR Reports - 2019 HMIS Tools 

2. The link also appears after database backup so find a way to force it to be just after to the UgandaEMR Reports link. I think we should rename it to UgandaEMR Reports - 2014 HMIS Tools

Comment (dbaluku) [reply omod/src/main/resources/apps/ugandaemr_reports_app.json]: Done

File: api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/FirstTimeHIVTESTConverter.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+public class FirstTimeHIVTESTConverter implements DataConverter {
+    @Override
+    public Object convert(Object obj) {
+
+        if (obj == null) {
+            return "";
+        }
+        Concept value = ((Obs)obj).getValueCoded();
+        if(value != null && value.equals(Dictionary.getConcept("1066AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"))) {
+            return "YES";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/FirstTimeHIVTESTConverter.java]: @dbaluku This should be Yes and No, not capitals

File: api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/TestedForMoreThanOnceInLast12MonthsConverter.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+public class TestedForMoreThanOnceInLast12MonthsConverter implements DataConverter {
+    @Override
+    public Object convert(Object obj) {
+
+        if (obj == null) {
+            return "NO";
+        }
+        Double value  = ((Obs)obj).getValueNumeric();
+        if(value != null && value > 1 ) {
+            return "YES";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/TestedForMoreThanOnceInLast12MonthsConverter.java]: @dbaluku This should be title case, not capital letters

File: api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/FirstTimeHIVTESTConverter.java (hunk 2 above)
Comment (dbaluku) [reply api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/FirstTimeHIVTESTConverter.java]: These are to be kept  capital  to have consistency in the report

**Pull request merged**